runtime/queries/ron/highlights.scm

```diff
@@ -1 +1,44 @@
-; inherits: rust
\ No newline at end of file
+; Literals
+;------------
+
+(string) @string
+(boolean) @constant.builtin.boolean
+(integer) @constant.numeric.integer
+(float) @constant.numeric.float
+(char) @constant.character
+
+; Structs
+;------------
+
+(enum_variant) @type.enum.variant
+(struct_entry (_) @variable.other.member ":")
+(struct_name (identifier)) @type
+
+
+
+
+; Comments
+;------------
+
+(line_comment) @comment.line
+(block_comment) @comment.block
+
+
+; Punctuation
+;------------
+
+"," @punctuation.delimiter
+":" @punctuation.delimiter
+
+"(" @punctuation.bracket
+")" @punctuation.bracket
+"[" @punctuation.bracket
+"]" @punctuation.bracket
+"{" @punctuation.bracket
+"}" @punctuation.bracket
+
+
+; Special
+;------------
+(escape_sequence) @constant.character.escape
+(ERROR) @error
```

Comment:
In general we have avoided adding this to queries. Eventually we want to make this a config option, so when the option is enabled this is automatically appended to the query text and when the config option is off we just use the queries untouched. For now let's remove this line since the error highlights tend to be really noisy when typing

Reply:
thats fair, I kept it in there because ron doesn't have an lsp so this will show places where there is an error but I can see how this adds extra noise